camdaemon.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import datetime
+import time
+
+import picamera2
+
+import config
+
+
+def picam2_setup(picam2: picamera2.Picamera2) -> None:
+    picam2.start_preview(picamera2.Preview.NULL)
+
+    preview_config = picam2.create_preview_configuration(main={"size": (800, 600)})
```

Comment:
love the explicit smaller size!  should make this go a lot faster

since we're adding a `config.py` with `.env` loader  - let's move the width and height to be a env var so it's easy to change at a later date and folks can customize

Reply:
Created #12 to let the user modify the displayed image via GET. Since this is just the internal preview image for calibrating exposure settings, not sure if the user would benefit from changing?